### PR TITLE
libpcap: add libpcap package

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: recipes-debian/libpcap/libpcap_debian.bb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+inherit debian-package
+require recipes-debian/sources/libpcap.inc
+
+DEBIAN_PATCH_TYPE = 'quilt'
+FILESPATH_append = ":${COREBASE}/meta/recipes-connectivity/libpcap/libpcap"
+
+SRC_URI += "file://0001-pcap-usb-linux.c-add-missing-limits.h-for-musl-syste.patch"
+
+BINCONFIG = "${bindir}/pcap-config"
```
Comment (masami256) [inline recipes-debian/libpcap/libpcap_debian.bb:19]: Poky's bb file contains [following line ](https://git.yoctoproject.org/cgit.cgi/poky/tree/meta/recipes-connectivity/libpcap/libpcap_1.9.0.bb?h=warrior#n19 ) , so it looks required to build this package.

```
inherit autotools binconfig-disabled pkgconfig bluetooth
```

File: recipes-debian/libpcap/libpcap_debian.bb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+SECTION = "libs/network"
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=5eb289217c160e2920d2e35bddc36453 \
+                    file://pcap.h;beginline=1;endline=32;md5=39af3510e011f34b8872f120b1dc31d2"
+DEPENDS = "flex-native bison-native"
+
+inherit debian-package
+require recipes-debian/sources/libpcap.inc
+
+DEBIAN_PATCH_TYPE = 'quilt'
```
Comment (masami256) [inline recipes-debian/libpcap/libpcap_debian.bb]: If  debian/source/format file says "3.0 (quilt) ", you don't have to set DEBIAN_PATCH_TYPE.

**Comment by yuwata** [comment]
@masami256 Thank you for the comments. I've completely rewritten the .bb file, now which is based on yocto-2.4, as warrior uses libpcap-1.9 but debian buster uses libpcap-1.8.1. PTAL.

**Comment by masami256** [comment]
> @masami256 Thank you for the comments. I've completely rewritten the .bb file, now which is based on yocto-2.4, as warrior uses libpcap-1.9 but debian buster uses libpcap-1.8.1. PTAL.

@yuwata That's nice! btw, yocto's QA check found following warning, could you check it please?

```
Initialising tasks: 100% |#############################################################################################################################################################| Time: 0:00:00
Sstate summary: Wanted 6 Found 0 Missed 6 Current 74 (0% match, 92% complete)
NOTE: Executing SetScene Tasks
NOTE: Executing RunQueue Tasks
WARNING: libpcap-1.8.1-r0 do_patch: Fuzz detected:

Applying patch libpcap-pkgconfig-support.patch
patching file Makefile.in
Hunk #1 succeeded at 69 with fuzz 2 (offset 8 lines).
Hunk #2 succeeded at 304 (offset 14 lines).
patching file configure.ac
Hunk #1 succeeded at 1814 (offset 9 lines).
patching file libpcap.pc.in

The context lines in the patches can be updated with devtool:

    devtool modify libpcap
    devtool finish --force-patch-refresh libpcap <layer_path>

Don't forget to review changes done by devtool!

WARNING: libpcap-1.8.1-r0 do_patch: QA Issue: Patch log indicates that patches do not apply cleanly. [patch-fuzz]
NOTE: Tasks Summary: Attempted 568 tasks of which 552 didn't need to be rerun and all succeeded.
```

**Comment by yuwata** [comment]
@masami256 Done. PTAL. Thank you.

**Comment by yuwata** [comment]
> yocto's QA check

BTW, how to invoke that?

**Comment by masami256** [comment]
> > yocto's QA check
> 
> BTW, how to invoke that?

@yuwata This warning came from patch task, so maybe easiest way is use "-f" option to run build or patch to force run patch task again. e.g. ``` bitbake -f libpcap -c patch ```

**Comment by yuwata** [comment]
Ah, I see. I thought there is a specific command to invoke that. Actually, I found the message by running a similar way: `bitbake libpcap -c cleanall; bitbake libpcap`. Thanks.

**Comment by masami256** [comment]
LGTM!

**Pull request merged**